src/WebformCivicrmPreProcess.php

```diff
@@ -143,6 +143,9 @@ public function alterForm() {
       $existing_component = $this->node->getElement("civicrm_{$c}_contact_1_contact_existing");
       // Search for contact if the user hasn't already chosen one
       if ($existing_component && empty($submitted_contacts[$c])) {
+        // When the contact has been manually selected/entered and the form has therefore been filled via
+        // an ajax request, we must make the selected contact_id value available to findContact()
+        $this->ent['contact'][$c]['id'] = $this->form_state->getUserInput()[$existing_component['#form_key']];
```

Comment:
@bsilvern Since this is in preprocess, `$existing_component['#form_key']` won't always be present in the user input. Can we enclose this in an if condition to avoid notices?

Reply:
@jitendrapurohit I'm not seeing any notices. Please explain the condition where $existing_component is not null but the '#form_key' element of that array is undefined.

Reply:
@jitendrapurohit Nevermind. I wasn't seeing the notification because I had PHP warnings disabled. I'll submit an updated PR shortly.

Reply:
@jitendrapurohit PR has been updated. Thanks for flagging this.